esmvaltool/recipes/recipe_combined_indices.yml

```
@@ -15,29 +15,36 @@ documentation:
     - c3s-magic
 
 datasets:
-  - {dataset: MPI-ESM-MR,  project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1950,  end_year: 2005}
-  #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
-  #- {dataset: IPSL-CM5A-MR, type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1981,  end_year: 2000}
+  - {dataset: MPI-ESM-MR,  project: CMIP5,  mip: Omon,  exp: historical,  ensemble: r1i1p1,  start_year: 1950,  end_year: 2005}
+  - {dataset: bcc-csm1-1,  project: CMIP5,  mip: Omon,  exp: historical,  ensemble: r1i1p1,  start_year: 1950,  end_year: 2005}
+
+
+preprocessors:
+  preprocessor1:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
 
-   
 diagnostics:
   combine_indices:
     description: Combine indices and/or compute area weighted averages.
     variables:
-      psl:
-        mip: Amon
+      tos:
+      #zg:
+        preprocessor: preprocessor1
+        mip: Omon
     scripts:
       main:
         script: magic_bsc/combined_indices.r
         ## Weights for combining indices - if not "equal" or "NULL", must be the same length as the number of indices
-        region: NAO #Nino3, Nino3.4, Nino4, NAO or SOI  
+        region: Nino3.4 #Nino3, Nino3.4, Nino4, NAO or SOI  
 
         # Compute running mean?
-        running_mean: 11 #3
+        running_mean: NULL  #3
```

```suggestion
        running_mean: null
```
please use `yamllint` to check that the recipe is according to the standards